src/main/java/com/faforever/client/main/event/LocalReplaysChangedEvent.java

```diff
@@ -0,0 +1,13 @@
+package com.faforever.client.main.event;
+
+import com.faforever.client.replay.Replay;
+import lombok.Data;
+
+import java.util.Collection;
+
+@Data
+public class LocalReplaysChangedEvent {
```

Comment:
Does data holder class represents `event`?

Reply:
I mean maybe we could give here better naming.

Reply:
Not sure what you mean. The `@Data` annotation is part of Lombok and automatically generates getters/setters etc.

Reply:
To be more precise, the event is a value type right? (Create once, read only)
In this case @Value would be the better lombok annotation as it makes the variables final and the object immutable.

Reply:
Changed.